src/hapi/pipelines/utilities/text_parsing.py

```diff
@@ -0,0 +1,27 @@
+from typing import Dict
+
+from hdx.location.names import clean_name
+from hdx.location.phonetics import Phonetics
+
+
+def get_code_from_name(
+    name: str, code_lookup: Dict[str, str], code_mapping: Dict[str, str]
```

Comment:
Could you add a docstring that explains the parameters? (code_lookup and code_mapping can be confusing).